groups/validators.py

```diff
@@ -0,0 +1,35 @@
+from django.utils import timezone
+from CultureAnalyzer.exceptions import PValidationError
```

Comment:
According to pep-8 a blank like should be added before imports from current project